### ISSUE TITLE
Glacier APIs, always throw exception "Required parameter missing: API version"

### ISSUE THREAD

**Issue by adam-fowler** [swift-aws/aws-sdk-swift#98]
It seems like all the Glacier code is throwing the same error. The following code 
```
do {
    let glacier = Glacier(region:.euwest2)
    let description = try glacier.describeVault(Glacier.DescribeVaultInput(vaultName:"vaultName", accountId:"-"))
    print(description)
} catch let error {
    print(error)
}
```
Always outputs the following error
```
AWSError(message: "Required parameter missing: API version", rawBody: "{\"code\":\"MissingParameterValueException\",\"message\":\"Required parameter missing: API version\",\"type\":\"Client\"}")
```
I've tried this with listVaults() and listJobs() with the same results

**Comment by jonnymacs** [comment]
Yeah. That’s good thinking.

I am thinking I’d like to avoid a service specific customization in the
core lib directly by implementing a glacier middleware. I have something
that should work, I’ll push up a branch when it’s ready (today / tonight)

**Comment by jonnymacs** [comment]
fixed in #102
